frontend/src/features/recommendation/RecommendationApi.tsx

```diff
@@ -56,6 +56,7 @@ export const getRecommendedVideos = async (searchString: string) => {
       offset: getNumberValue('offset'),
       search: params.get('search') ?? undefined,
       language: params.get('language') ?? undefined,
+      uploader: encodeURI(params.get('uploader') ?? ''),
```

Comment:
The generated client implements the necessary function to encode these params correctly. So the values should not be encoded here either.

You can have a look at the `getQueryString()` function in "services/openapi/core.request.ts" if you want to learn more about how it works.